tutorial/source/cjs.rst

```diff
@@ -0,0 +1,11 @@
+Capture-Recapture Models (CJS Models)
+===================
```

Comment:
Does this underbar need to be full length? I've seen sphinx complain before.

Reply:
done